apps/prairielearn/src/ee/pages/institutionAdminGeneral/institutionAdminGeneral.html.ts

```diff
@@ -55,6 +58,7 @@ export function InstitutionAdminGeneral({
           navbarType: 'institution',
           navPage: 'institution_admin',
           navSubPage: 'general',
+          lti13_enabled: false,
```

Comment:
Why is this set?

Reply:
This was leftover from trying to debug when the navbar loaded with an error of `lti13_enabled is undefined`. This was before I knew to enable `isEnterprise` and everything loaded appropriately. This has now been removed in commit [fb604ae](https://github.com/PrairieLearn/PrairieLearn/pull/9045/commits/fb604aec1a5e82dbf069cab10a5723c80a85a5c8).